src/gallery/index.jsx

```diff
@@ -49,15 +49,15 @@ const Gallery = () => {
       state: false,
     },
   ];
-  const stateArray = galleryArray.filter((value) => value.state);
+  // const stateArray = galleryArray.filter((value) => value.state);
```

Comment:
I think you won't need this, you should remove it instead and also remove the state props from the items of the array